tools/swagger_codegen/py-client/templates/configuration.mustache

```diff
@@ -0,0 +1,259 @@
+# coding: utf-8
+
+{{>partial_header}}
+
```

Comment:
Where is this template getting used ?

Reply:
It is used for swagger client sdk code generation -- https://github.com/Azure/sonic-mgmt-framework/pull/54/files#diff-34e4d2294b651f9e11e26c8baa128658R133.